src/components/tabs/FunctionsTab.vue

```diff
@@ -166,10 +167,12 @@ export default class FunctionsTab extends Vue {
 
       // Parse file - report any errors
       const parsed: Result<ParsedFunction[]> = parse(event.target.result as string);
-      if ('message' in parsed) {
+      if (parsed instanceof Error) {
```

Comment:
I'm not sure that `instanceOf` works, because Javascript isn't great at runtime type checking with interfaces. I'm pretty sure @gidonkatten tried it, too.

Reply:
Didn't realise you could do this but it seems to work for me

Reply:
Hmm, would you rather change it for safety or should we leave it as it is then?

Reply:
I think what you have done is better